src/main/java/io/codelex/loops/practice/MultiplyByItself.java

```diff
@@ -0,0 +1,26 @@
+package io.codelex.loops.practice;
+
+import java.util.Scanner;
+
+public class MultiplyByItself {
+
+    public static void main(String[] args) {
+        int i, n;
+
+        System.out.print("Input number : ");
+        Scanner in = new Scanner(System.in);
+        i = in.nextInt();
+        System.out.print("Input number of terms : ");
+        Scanner in2 = new Scanner(System.in);
+        n = in2.nextInt();
+        long result = 1;
+
+
+
+        for (; n != 0; --n) {
```

Comment:
Šis pieraksts ir mulsinošs, tehniski tas strādā, bet faktiski tā neraksta, labāk lietot pilno if formu šādi:
`for (j=n; j != 0; j--) {`